processors/validator.py

```diff
@@ -6,13 +6,14 @@
 
 REQUIRED_FIELDS = [
     'award.awardingAgency.officeCode',
-    'award.actionDate'
+    'award.awardees.businessName',
+    'award.awardees.businessAddress',
     ]
 LENGTHS = [('award.awardingAgency.officeCode', 6)]
 NUMERIC_FIELDS = ['transaction.programActivity']
 POSSIBLE_VALUES = [
     ('transaction.objectClass', ['4110', '4101']),
```

Comment:
Yes, this is a subset for the pilot since those codes are particular to grants and Outlays.